### ISSUE TITLE
Extension error: TypeError: undefined is not a valid URL.

### ISSUE THREAD

**Issue by wellington1993** [EFForg/https-everywhere#16562]
Type: other
Domain: ? (Maybe: https://open.scdn.co/static/web-player.28147008.js)
Confirmed Domain: https://github.com/tensorflow/tensorflow

*The error occurs in a moment of opening page when switch the tab(After the page restore because Firefox crashed)*

I'm being have many errors for that add on in my browser console.
Sometimes I clean and remove extensions that generate errors.
But I like the proposal of this and reporting this error below.

The add on error:
```
[[Exception stack
onBeforeRequest@moz-extension://877f5946-3af1-4a70-bf22-85a142b651d4/background-scripts/background.js:335:24
Current stack
applySafeWithoutClone@resource://gre/modules/ExtensionCommon.jsm:529:133
fire@resource://gre/modules/ExtensionChild.jsm:979:28
receiveMessage@resource://gre/modules/ExtensionChild.jsm:982:38
_callHandlers/<@resource://gre/modules/MessageChannel.jsm:876:29
_callHandlers@resource://gre/modules/MessageChannel.jsm:875:14
_handleMessage/deferred.promise<@resource://gre/modules/MessageChannel.jsm:952:7
_handleMessage@resource://gre/modules/MessageChannel.jsm:949:24
receiveMessage/<@resource://gre/modules/MessageChannel.jsm:218:9
receiveMessage@resource://gre/modules/MessageChannel.jsm:211:5
MessageListener.receiveMessage*FilteringMessageManager@resource://gre/modules/MessageChannel.jsm:201:5
get@resource://gre/modules/MessageChannel.jsm:436:14
addListener@resource://gre/modules/MessageChannel.jsm:763:7
ExtensionGlobal@jar:file:///home/wellington/softwares/firefox/omni.ja!/components/extension-process-script.js:58:5
init/<@jar:file:///home/wellington/softwares/firefox/omni.ja!/components/extension-process-script.js:113:42
@data:,
      Components.utils.import("resource://gre/modules/Services.jsm");

      Services.obs.notifyObservers(this, "tab-content-frameloader-created", "");
    :1:76
]]

```

Complete error:

```
JavaScript error: , line 0: EvalError: call to eval() blocked by CSP
JavaScript error: https://open.scdn.co/static/web-player.28147008.js, line 1: Error: AdManager: No accessToken provided
JavaScript error: https://open.scdn.co/static/web-player.28147008.js, line 1: Error: Fatal error dialog: REQUEST_ME_FAILURE
JavaScript error: resource://gre/modules/WebNavigationContent.js, line 215: NS_NOINTERFACE: Component returned failure code: 0x80004002 (NS_NOINTERFACE) [nsIWebProgress.DOMWindow]
JavaScript error: resource://gre/modules/WebNavigationContent.js, line 215: NS_NOINTERFACE: Component returned failure code: 0x80004002 (NS_NOINTERFACE) [nsIWebProgress.DOMWindow]
Extension error: TypeError: undefined is not a valid URL. moz-extension://877f5946-3af1-4a70-bf22-85a142b651d4/background-scripts/background.js 335
[[Exception stack
onBeforeRequest@moz-extension://877f5946-3af1-4a70-bf22-85a142b651d4/background-scripts/background.js:335:24
Current stack
applySafeWithoutClone@resource://gre/modules/ExtensionCommon.jsm:529:133
fire@resource://gre/modules/ExtensionChild.jsm:979:28
receiveMessage@resource://gre/modules/ExtensionChild.jsm:982:38
_callHandlers/<@resource://gre/modules/MessageChannel.jsm:876:29
_callHandlers@resource://gre/modules/MessageChannel.jsm:875:14
_handleMessage/deferred.promise<@resource://gre/modules/MessageChannel.jsm:952:7
_handleMessage@resource://gre/modules/MessageChannel.jsm:949:24
receiveMessage/<@resource://gre/modules/MessageChannel.jsm:218:9
receiveMessage@resource://gre/modules/MessageChannel.jsm:211:5
MessageListener.receiveMessage*FilteringMessageManager@resource://gre/modules/MessageChannel.jsm:201:5
get@resource://gre/modules/MessageChannel.jsm:436:14
addListener@resource://gre/modules/MessageChannel.jsm:763:7
ExtensionGlobal@jar:file:///home/wellington/softwares/firefox/omni.ja!/components/extension-process-script.js:58:5
init/<@jar:file:///home/wellington/softwares/firefox/omni.ja!/components/extension-process-script.js:113:42
@data:,
      Components.utils.import("resource://gre/modules/Services.jsm");

      Services.obs.notifyObservers(this, "tab-content-frameloader-created", "");
    :1:76
]]
JavaScript error: , line 0: EvalError: call to eval() blocked by CSP
JavaScript error: , line 0: EvalError: call to eval() blocked by CSP

```
Thanks!

**Comment by pipboy96** [comment]
I don't see any errors and there are definitely no crashes.
